renovate.json

```diff
@@ -2,5 +2,19 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     "config:base"
+  ],
+  "regexManagers": [
+    {
+      "fileMatch": ["^Makefile$"],
+      "matchStrings": ["FLAGD_VERSION=(?<currentValue>.*?)\\n"],
+      "depNameTemplate": "open-feature/flagd",
+      "datasourceTemplate": "github-releases"
+    },
+    {
+      "fileMatch": ["^README.md$"],
```

Comment:
Admittedly, I haven't manually tested that this matcher works but I can't see why it wouldn't given that the other does